.vscode/launch.json

```diff
@@ -30,33 +30,6 @@
             "console": "integratedTerminal",
             "smartStep": true,
         },
-        {
-            "type": "node",
-            "request": "launch",
-            "name": "Matrix Tests",
-            "cwd": "${workspaceFolder}/packages/dds/matrix",
-            "program": "${workspaceFolder}/node_modules/mocha/bin/_mocha",
-            "args": [
-                "-r",
-                "ts-node/register",
-                "-r",
-                "source-map-support/register",
-                "--no-timeouts",
-                // "test/**/matrix.spec.ts",
-                "test/**/matrix.undo.spec.ts",
-                // "test/**/matrix.stress.spec.ts",
-                // "test/**/sparsearray2d.spec.ts",
-                // "-f",
-                // "fail",
-                "--exit"
-            ],
-            "env": {
-                "TS_NODE_PROJECT": "tsconfig.json",
-            },
-            // "skipFiles": ["<node_internals>/**"],
-            "console": "integratedTerminal",
-            "smartStep": true,
-        },
```

Comment:
The general 'Debug Mocha Tests' tasks is now good enough that I don't need this.